bundles/binding/org.openhab.binding.zwave/database/aeon/zw096.xml

```diff
@@ -205,7 +204,6 @@
             <Type>byte</Type>
             <Default>505050</Default>
```

Comment:
Wow, so this is another instance that just leaves me scratching my head.  The value is a size 3, and is the brightness in 1-100% for each color.  Here is what I will do, and you tell me if it makes sense.  The default is 50% brightness across each, so that equates to 505050, or 323232 if hex.  What they don't say is if 0x32, which is 50 decimal is the 50% number of if 256/2 = 128 = 0x80 is the 50% number.  Let's assume the lower to be safe, and see if we can find out from Aeon what the real scoop is.

So...  323232 equates to decimal 3289650 and I will make that the default value.
and .. 636363 equates to decimal 6513507 and that will be the max.

As noted previously, this smells bad.